crates/interpreter/src/side_table.rs

```diff
@@ -67,15 +60,15 @@ impl<'m> Metadata<'m> {
     }
 
     pub fn branch_table(&self) -> &[BranchTableEntry] {
-        bytemuck::cast_slice(&self.0[5 ..])
+        bytemuck::cast_slice(&self.0[10 ..])
     }
 
     pub fn parser_range(&self) -> Range<usize> {
-        self.read_u32(1) .. self.read_u32(3)
+        self.read_u32(2) .. self.read_u32(6)
     }
 
     fn read_u32(&self, idx: usize) -> usize {
-        bytemuck::pod_read_unaligned::<u32>(bytemuck::cast_slice(&self.0[idx .. idx + 2])) as usize
+        bytemuck::pod_read_unaligned::<u32>(bytemuck::cast_slice(&self.0[idx .. idx + 4])) as usize
```

Comment:
Same here, we can use `u32::from_le_bytes()`. We probably don't need bytemuck anymore.